README.md

```diff
@@ -49,16 +59,23 @@ or
 
 
 Documentation
-----------
+-------------
 Please check out the [Read The Docs documentation](https://screenpy-appium-docs.readthedocs.io/en/latest/) for the latest information about this module!
 
+You can also read the [ScreenPy Docs](https://screenpy-docs.readthedocs.io/en/latest/) for more information about ScreenPy in general.
```

Comment:
I think this is a good idea—i do link to the ScreenPy docs on the first page of all of the extension docs, but maybe they want to start there from here.

Reply:
hehe.. something something.. cruft..